connectors/sources/sharepoint_online.py

```diff
@@ -1280,6 +1289,21 @@ def _is_site_admin(user):
                 if email:
                     user_access_control.add(_prefix_email(email))
 
+            if is_sharepoint_group(user):
+                site_group_id = user.get("id")
+                async for site_group_user in self.client.site_groups_users(
+                    site["webUrl"], site_group_id
+                ):
+                    site_group_user_name = site_group_user.get(
+                        "UserPrincipalName", None
```

Comment:
super nit, but `None` is already the default value for a `get()` where the key doesn't exist, so you don't need to provide it as an explicit arg.